tests/utils.py

```diff
@@ -19,7 +23,45 @@ def unorder(value):
     return unorder(xmltodict.parse(xml))
 
 
+NAMESPACE_ATTRIBUTE = re.compile(' xmlns="[^"]+"', flags=re.UNICODE)
```

Comment:
Also, there is some hypothetical case where the xml attribute can be something like - 
```xml
...
<tag xmlns="..."
...
```
 Should we worry about it ?